src/main/java/de/tum/in/www1/artemis/domain/LearningGoal.java

```diff
@@ -25,16 +25,20 @@
 
     @ManyToOne
     @JoinColumn(name = "course_id")
+    @JsonIgnoreProperties("learningGoals")
     private Course course;
 
+    // ToDo remove this connection, only go through lecture units
```

Comment:
is this TODO for a later PR?

Reply:
Done in : c70aa0f01060fd36a9f540c34c4b5f0e009a4410